client/src/enums/entity-type-code.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Enum for Entity type codes.
+ * Used for mapping conversions to support namex ui's split entityTypes.
+ * */
+const EntityTypeCode = {
+  GP: 'GP', // General Partnership
+}
+
+export default Object.freeze(EntityTypeCode)
```

Comment:
Is freeze needed for a const?

Actually, this isn't an enum at all, is it? Kind of a dictionary object I guess?

Reply:
I know eh! I didn't actually know what to do here 😆 

It appeared this was a way to prevent re-assignment of these values in es5, this is the closest equivalent i could find to what TS does with `export enum`. 

🤷  Open to any and all ideas, this was just the closest i could find that worked without issue haha

Reply:
To clarify further... const prevents re assignment where the freeze prevents mutation on the inner values themselves.

Or so i've read 🤷

Reply:
Interesting though! 
It's not until we have to work backwards a little (using js and es5) where we see why some decisions and helpers were made in ts and es6 😁

Reply:
Do nothing here.
Thanks for the explanation. Makes sense.
Yep, it will be nice to upgrade this to ES6 and TS :)